akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -1583,6 +1583,10 @@ abstract class GraphStageLogic private[stream] (val inCount: Int, val outCount:
     override def toString = s"SubSourceOutlet($name)"
   }
 
+  override def toString: String = {
+    attributes.get[Attributes.Name].map(attr => s"${getClass.toString}-${attr.n}").getOrElse(getClass.toString)
+  }
```

Comment:
Usually with a single expression method body we leave out the parenthesises.

Reply:
Thanks, @johanandren. I have removed the parenthesis.